### PR TITLE
fix for --help not working

### PR DESCRIPTION
**Pull request by 1ntEgr8**
the --help command was not working when creating a fresh clone of this repo. 

rearranged the execution of `showHelp` as a fix

### CODE REVIEW

File: packages/@romejs/cli-flags/Parser.ts [hunk 1]
```diff
@@ -328,6 +328,7 @@ export default class Parser<T> {
   async init(): Promise<T> {
     // Show help for --version
     if (this.flags.has('version')) {
+      console.log(this.flags);
```
Comment (diokey) [inline packages/@romejs/cli-flags/Parser.ts]: Remove console.log

**Comment by 1ntEgr8** [comment]
I removed the .gitignore; saw something in the discord chat about not wanting one for this repo

File: packages/@romejs/cli-flags/Parser.ts [hunk 2]
```diff
@@ -355,19 +362,9 @@ export default class Parser<T> {
       }
 
       consumer.enforceUsedProperties('flag', false);
-      this.currentCommand = undefined;
```
Comment (sebmck) [inline packages/@romejs/cli-flags/Parser.ts]: Why was this line removed?

Comment (1ntEgr8) [reply packages/@romejs/cli-flags/Parser.ts]: shouldn't have been removed. will check diffs before pushing next time oops.

File: packages/@romejs/cli-flags/Parser.ts [hunk 3]
```diff
@@ -336,6 +336,13 @@ export default class Parser<T> {
 
     let definedCommand: undefined | DefinedCommand;
 
+    if (this.helpMode) {
```
Comment (sebmck) [inline packages/@romejs/cli-flags/Parser.ts]: This actually breaks focusing help on a command because `definedCommand` is never initialized. eg. `rome parse --help`. The actual fix for this is to wrap `enforceUsedProperties` with `if (!this.helpMode)`.

**Comment by sebmck** [comment]
Superseded by #262.
